jdk/test/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java

```diff
@@ -484,6 +484,28 @@
  * @run main/othervm -Djava.security.debug=certpath CAInterop globalsigne46 CRL
  */
 
+/*
+ * @test id=ssltlsrootecc2022
+ * @bug 8341057
+ * @summary Interoperability tests with SSL TLS 2022 root CAs
+ * @library /test/lib
+ * @build jtreg.SkippedException ValidatePathWithURL CAInterop
+ * @run main/othervm/manual -Djava.security.debug=certpath,ocsp CAInterop ssltlsrootecc2022 DEFAULT
+ * @run main/othervm/manual -Djava.security.debug=certpath,ocsp -Dcom.sun.security.ocsp.useget=false CAInterop ssltlsrootecc2022 DEFAULT
+ * @run main/othervm/manual -Djava.security.debug=certpath CAInterop ssltlsrootecc2022 CRL
```

Comment:
In 8u we usually run those tests (not specifying them as manual). That's why they weren't run in GHA. Please remove `/manual`.

Reply:
Ah yes, good catch. This is going to be a pain with these backports going forward.